src/setupTests.tsx

```diff
@@ -25,6 +25,39 @@ window.print = jest.fn(() => {
 
 const printMock = mockOf(window.print)
 
+function mockSpeechSynthesis() {
+  const w = window as {
+    speechSynthesis: typeof speechSynthesis
+    SpeechSynthesisUtterance: typeof SpeechSynthesisUtterance
+    SpeechSynthesisEvent: typeof SpeechSynthesisEvent
+  }
+
+  w.speechSynthesis = makeSpeechSynthesisDouble()
+  w.SpeechSynthesisUtterance = jest.fn().mockImplementation(text => ({ text }))
+  w.SpeechSynthesisEvent = jest.fn()
+}
+
+function makeSpeechSynthesisDouble(): typeof speechSynthesis {
+  return {
+    addEventListener: jest.fn(),
+    cancel: jest.fn(),
+    dispatchEvent: jest.fn(),
+    getVoices: jest.fn(),
+    onvoiceschanged: jest.fn(),
+    pause: jest.fn(),
+    paused: false,
+    pending: false,
+    removeEventListener: jest.fn(),
+    resume: jest.fn(),
+    speak: jest.fn(),
+    speaking: false,
```

Comment:
It'd be great if all of this was just part of `jsdom`, but [a search for `speechSynthesis` in that repo literally returns nothing](https://github.com/jsdom/jsdom/search?q=speechsynthesis&unscoped_q=speechsynthesis).

Reply:
It's lonely on the bleeding edge of A11y. 🤦🏼‍♂️